src/remediations/__snapshots__/fifi.integration.js.snap

```diff
@@ -75,6 +75,73 @@ exports[`FiFi connection status get connection status with false smartManagement
 }"
 `;
 
+exports[`FiFi connection status get connection status with smartManagment but not enabled with config manager 1`] = `
+"{
+    \\"meta\\": {
+        \\"count\\": 7,
+        \\"total\\": 7
+    },
+    \\"data\\": [
+        {
+            \\"endpoint_id\\": null,
+            \\"executor_id\\": \\"01bf542e-6092-485c-ba04-c656d77f988a\\",
+            \\"executor_type\\": \\"satellite\\",
+            \\"executor_name\\": \\"Satellite 01bf542e-6092-485c-ba04-c656d77f988a\\",
+            \\"system_count\\": 4,
+            \\"connection_status\\": \\"no_source\\"
+        },
+        {
+            \\"endpoint_id\\": \\"805\\",
+            \\"executor_id\\": \\"722ec903-f4b5-4b1f-9c2f-23fc7b0ba390\\",
+            \\"executor_type\\": \\"satellite\\",
+            \\"executor_name\\": \\"Satellite 1 (connected)\\",
+            \\"system_count\\": 3,
+            \\"connection_status\\": \\"connected\\"
+        },
+        {
+            \\"endpoint_id\\": \\"806\\",
+            \\"executor_id\\": \\"409dd231-6297-43a6-a726-5ce56923d624\\",
+            \\"executor_type\\": \\"satellite\\",
+            \\"executor_name\\": \\"Satellite 2 (disconnected)\\",
+            \\"system_count\\": 6,
+            \\"connection_status\\": \\"disconnected\\"
+        },
+        {
+            \\"endpoint_id\\": null,
+            \\"executor_id\\": \\"72f44b25-64a7-4ee7-a94e-3beed9393972\\",
+            \\"executor_type\\": \\"satellite\\",
+            \\"executor_name\\": \\"Satellite 3 (no receptor configured)\\",
+            \\"system_count\\": 5,
+            \\"connection_status\\": \\"no_receptor\\"
+        },
+        {
+            \\"endpoint_id\\": \\"808\\",
+            \\"executor_id\\": \\"63142926-46a5-498b-9614-01f2f66fd40b\\",
+            \\"executor_type\\": \\"satellite\\",
+            \\"executor_name\\": \\"Satellite 4 (connected)\\",
+            \\"system_count\\": 1,
+            \\"connection_status\\": \\"connected\\"
+        },
+        {
+            \\"endpoint_id\\": null,
+            \\"executor_id\\": null,
+            \\"executor_type\\": null,
+            \\"executor_name\\": null,
+            \\"system_count\\": 1,
+            \\"connection_status\\": \\"no_executor\\"
+        },
+        {
+            \\"endpoint_id\\": null,
+            \\"executor_id\\": null,
+            \\"executor_type\\": \\"RHC\\",
+            \\"executor_name\\": null,
+            \\"system_count\\": 2,
+            \\"connection_status\\": \\"disabled\\"
```

Comment:
Even though this is reported as disabled to the user, if they submit the pre-flight will remediations still attempt to execute playbook on these hosts? If not, pls point me to the code that takes care about that.

Reply:
Hey!  so yes.  the "execute" method will only run the executors marked with 'connected'

--> right here we filter out all NON connected executors https://github.com/RedHatInsights/insights-remediations/blob/master/src/remediations/fifi.js#L140

Reply:
I see, thanks!